src/instruments/src/PFD/SpeedIndicator.tsx

```diff
@@ -578,8 +546,8 @@ class VLsBar extends DisplayComponent<{ bus: EventBus }> {
             this.setVlsPath();
         });
 
-        sub.on('vls').handle((vls) => {
-            this.vls = this.smoothSpeeds(vls);
+        sub.on('vLs').handle((vls) => {
```

Comment:
`withArinc429Precision` maybe? Not mandatory IMO, but if it won't hurt, I'd add it :)